conmon/Makefile

```diff
@@ -1,8 +1,13 @@
+include ../Makefile.inc
+
 src = $(wildcard *.c)
 obj = $(src:.c=.o)
 
 override LIBS += $(shell pkg-config --libs glib-2.0)
-override CFLAGS += -std=c99 -Os -Wall -Wextra $(shell pkg-config --cflags glib-2.0)
+
+VERSION = $(shell sed -n -e "/const Version/ s|const Version = ||p" < ../version/version.go)
```

Comment:
```console
$ sed -n -e "/const Version/ s|const Version = ||p" version/version.go 
"1.9.0-dev"
```

I don't think you want those quotes (you're adding them again below).  How about:

```makefile
VERSION = $(shell sed -n -e 's/^const Version = "\([^"]*\)"/\1/p' ../version/version.go)
```

Reply:
I had the simpler sed statement (and the additional quotes) as anyway `make` would take care of the quotes.  I amended the patch to use your version.